src/screens/Set_Password/set_password.jsx

```diff
@@ -0,0 +1,83 @@
+import { Formik, Field } from "formik";
```

Comment:
Remove this file too, copy it elsewhere, we can create a new branch for this feature too